tailscale-hosts.sh

```diff
@@ -10,7 +10,7 @@ get_ts_hosts() {
   # Also, we do filter out items that have an empty DNSName which allows us to
   # skip services (eg: hello.ipn.dev)
   tailscale status --json | \
-    jq -r '[.Self, .Peer[]] |
+    jq -r '[.Peer[]] |
```

Comment:
Thanks for giving this a try :)

I don't understand the reasoning behind your PR though.
Removing `Self` will only leave the host of the DNS server unresolvable for its clients.

Reply:
Hi,

My understanding is that `.Self` is the host itself and that this script would be run on a per machine basis. Therefore the machines' own hostname should be resolved with `127.0.0.1` and not the tailscale address.

My use-case was to use this instead of magic DNS to resolve hostnames since I need the DNS to resolve other hosts.

This is just a minor change that mostly applied to my use-case and if you feel this change is unwanted then by all means reject this pull request!

Reply:
Yes, `Self` is the current TS host.

This script's main purpose was to be run only on the host that's providing DNS, eg an OpenWRT router that runs dnsmasq/kresd. All hosts that using said DNS server would then be able to resolve tailscale hosts, regardless if they are themselves connected to the TS network or not.

That aside, I'm not strictly against supporting your use-case - just make it an optional thing ;)
I'm fine with a `--ignore-self` flag or something.